auth/src/main/res/values/colors.xml

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="authui_colorPrimary">#3E50B4</color>
-    <color name="authui_colorPrimaryDark">#2F3E9E</color>
-    <color name="authui_colorAccent">#3E50B4</color>
-    <color name="authui_inputTextColor">#FFFFFF</color>
-    <color name="authui_inputTextColorDark">#000000</color>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
```

Comment:
I do think your new colors look better, however I think the original color schemes were set by the person who made the UX mocks so I am hesitant to change them based on our personal preference.

I'll see if someone can weigh in on that.  But warning I may have to ask you to revert the colors, which will make your beautiful table obsolete :-/